### ISSUE TITLE
Howto - Meteor 1.6.1.1 + Angular 6 + Ionic4

### ISSUE THREAD

**Issue by atao60** [runisland/meteor-ionic4#1]
As I don't know where else to ask, I do it here.

I tried to merge [abritopach/angular-ionic-master-detail](https://github.com/abritopach/angular-ionic-master-detail) with Meteor 1.6.1.1 using your package. I failed, see [Meteor 1.6 + Angular 6 + ionic4](https://stackoverflow.com/questions/50574612/meteor-1-6-angular-6-ionic4).

I think I missed something obvious but I have no clue what it can be.

Any help would be welcome on how to set up a simple Meteor 1.6.1.1 + Ionic4 app, if possible with Angular 6.

**Comment by ghybs** [comment]
Hi,

Sorry I have not been using Meteor + Angular since a while now, but you can find documentation on https://angular-meteor.com/

From what I remember, the idea is either to:
- use Meteor CLI with [`angular-compilers`](https://atmospherejs.com/meteor/angular-compilers) package. You can then include Ionic.
- use Angular or Ionic CLI and integrate with Meteor through [`meteor-client-bundler`](https://github.com/Urigo/meteor-client-bundler)

If you want to use the 1st option (as you seem to have done), you should proceed incrementally, adding packages gradually to make sure you properly use and configure them, instead of trying to reproduce a full project at once, when it was not designed to be integrated with Meteor.

Otherwise you might have more luck using the 2nd option, which would probably be easier to integrate into a full Angular / Ionic project.

That being said, you can use Ionic4 without Angular, if you are not bound to Angular.
You can use Blaze, Vue or React.
Simply use Ionic's custom `<ion-**>` tags in your templates, this `runisland:ionic4` package, and these custom tags will be automatically replaced by Ionic4 in your browser.

Good luck!

**Comment by atao60** [comment]
Thank you for your quick answer.

Would you be interested in two enhancements:
- npm scripts in package.json ready to be used under Windows
- ability to override a local copy of "meteor-ionic4"

**Comment by atao60** [comment]
About local override. I'm using your package to import \@ ionic/angular  as an atmosphere package in a Meteor project  (when I import it as npm package, Meteor is no more able to get any npm package at runtime). To do it I used the instructions given by [Overriding published packages with a local version](https://guide.meteor.com/writing-atmosphere-packages.html#overriding-atmosphere-packages). But it didn't work as in package.js, the path './dist' is not right in such a context.

About Windows. We're two developpers, me working under Linux :-D, but the other one under Windows. We had to change the npm scripts for him to be able to work on the Meteor project with the local copy of meteor-ionic4.
